specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2021-11-01-preview/examples/GetMarketplaceRegistrationDefinitions.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "scope": "subscription/0afefe50-734e-4610-8a82-a144ahf49dea",
+    "$filter": "planIdentifier eq 'publisher.offerIdentifier.planName.version'",
```

Comment:
>publisher.offerIdentifier.planName.version [](http://example.com/codeflow?start=35&length=42)

would it be good if these match with the plan details in the response ?

Reply:
Generally I try to avoid changing existing examples but I guess it wouldn't be a bad idea as we are using GUIDs instead of placeholders